src/script/util/EmojiUtil.ts

```diff
@@ -61,16 +61,16 @@ Object.keys(emojiesList).forEach(key => {
   const emojiObject = emojiValue[0];
   const emojiNames = emojiObject.n;
 
-  emojiDictionary[key] = emojiNames[emojiNames.length - 1].replaceAll('-', ' ');
+  emojiDictionary.set(key, emojiNames[0].replaceAll('-', ' '));
 });
 
 export const getEmojiTitleFromEmojiUnicode = (emojiUnicode: string): string => {
-  return emojiDictionary[emojiUnicode];
+  return emojiDictionary.get(emojiUnicode) ?? '';
```

Comment:
Hmm.. I'm thinking on it: 
```suggestion
  return emojiDictionary.has(emojiUnicode) ? emojiDictionary.get(emojiUnicode) : '';
```

Need to compare performance of get and has :)